bootstrap-joining-demo/local/README.md

```diff
@@ -0,0 +1,23 @@
+# Local bootstrap demo
+
+To run the following need to resolve:
+
+```
+127.0.0.1   akka1
+127.0.0.2   akka2
+127.0.0.3   akka3
+```
```

Comment:
Isn't the intended use more that you have a single hostname resolve to multiple addresses?

The docs say:

> In the simplest version, we query for a service name – which each cluster manager, such as Kubernetes, Mesos or others define using their own naming schemes, and expect to get back a list of IPs that are related to this service.

and

> The simplest way of resolving multiple hosts of a (micro-)service is to perform a DNS lookup and treat all returned A records as hosts of the same service cluster.

With this configuration, akka1 is a single point of failure.

I know this is just meant as an example, but many users will imitate examples very literally.

Reply:
Yeah I wanted to do this but it would require running a DNS server locally to do an A record with multiple IPs which defeated my goal of a super simple one to be able to run without deps.

Reply:
Oh, that's a shame... I didn't realize that `/etc/hosts` doesn't support multiple addresses for the same host.

Maybe it would be more applicable for more people to have an example specifically designed to run in `minikube`.

Reply:
or with `docker-compose`? if you have kubernetes we'd want to suggest against using DNS and use the k8s API instead...

Reply:
Im very reluctant to add another demo (we already have 4, one of which is a kubernetes one you could run with minikube) that requires dependencies. I just wanted something you can run without installing things. Perhaps we should just be very clear in the readme about this demos limitation and purpose or not include it.

Reply:
That's fair enough... I think the nature of cluster bootstrap requires supporting infrastructure though. You would never use this method for a real application unless you also have a properly configured DNS server. I think either of your suggestions (clarify the readme or drop it) would help.